composer.json

```diff
@@ -8,17 +8,17 @@
         "ibexa/experience": "^3.3",
         "ezsystems/apache-tika-bundle": "^2.0",
         "ezsystems/comment-bundle": "^3.1",
-        "ezsystems/ezcommerce-admin-ui": "^1.2@dev",
-        "ezsystems/ezcommerce-base-design": "^3.3@dev",
-        "ezsystems/ezcommerce-checkout": "^1.0@dev",
-        "ezsystems/ezcommerce-erp-admin": "^3.3@dev",
-        "ezsystems/ezcommerce-fieldtypes": "^1.2@dev",
-        "ezsystems/ezcommerce-order-history": "^3.3@dev",
-        "ezsystems/ezcommerce-page-builder": "^1.2@dev",
-        "ezsystems/ezcommerce-price-engine": "^3.3@dev",
-        "ezsystems/ezcommerce-shop": "^3.3@dev",
-        "ezsystems/ezcommerce-shop-ui": "^1.1@dev",
-        "ezsystems/ezmigrationbundle": "^1.0@dev",
+        "ezsystems/ezcommerce-admin-ui": "^1.2.x-dev",
+        "ezsystems/ezcommerce-base-design": "^3.3.x-dev",
+        "ezsystems/ezcommerce-checkout": "^1.0.x-dev",
+        "ezsystems/ezcommerce-erp-admin": "^3.3.x-dev",
+        "ezsystems/ezcommerce-fieldtypes": "^1.2.x-dev",
+        "ezsystems/ezcommerce-order-history": "^3.3.x-dev",
+        "ezsystems/ezcommerce-page-builder": "^1.2.x-dev",
+        "ezsystems/ezcommerce-price-engine": "^3.3.x-dev",
+        "ezsystems/ezcommerce-shop": "^3.3.x-dev",
+        "ezsystems/ezcommerce-shop-ui": "^1.1.x-dev",
+        "ezsystems/ezmigrationbundle": "^1.0.x-dev",
```

Comment:
This can be `^1.0` as this is just a fork of kaliop migration and we don't really update it.